delfin/drivers/netapp/dataontap/cluster_mode.py

```diff
@@ -88,3 +88,6 @@ def collect_perf_metrics(self, context, storage_id,
     @staticmethod
     def get_capabilities(context, filters=None):
         return NetAppHandler.get_capabilities(filters)
+
+    def get_latest_perf_timestamp(self):
```

Comment:
![image](https://user-images.githubusercontent.com/12843055/147446812-b5d58106-94f5-4380-acfd-71e48aed1d9c.png)
Please add supplementary parameters

Reply:
The modified